tests/test_api.py

```diff
@@ -336,16 +343,23 @@ def test_re_purl_filter(client, api_path):
 
 
 def test_nvr_nevra_filter(client, api_path):
-    c1 = ComponentFactory(type=Component.Type.RPM, meta_attr={"epoch": "1"}, name="autotrace-devel")
+    c1 = ComponentFactory(
+        type=Component.Type.RPM,
+        meta_attr={"epoch": "1"},
+        name="autotrace-devel",
+        version="3.2.1",
+        release="1.0.1e",
+        arch="noarch",
```

Comment:
Does arch="noarch" make sense for RPMs? I thought that was only for CONTAINER_IMAGEs. Should it be arch="src" instead?

Reply:
It does make sense; most Python packages for example are not arch-specific so use noarch. Check out e.g. Brew build 1399682.